app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -19,19 +19,23 @@ def confirm_request
     render locals: { hide_auth_buttons: true }
   end
 
+  def persist_auth_token(auth)
```

Comment:
Are we able to make this method private since it does not need to be public and is not a controller action?